script/prerender/contentPages.tsx

```diff
@@ -194,7 +196,11 @@ interface Options {
   modules: string[];
   title: string;
 }
-function injectHTML(html: string, {body, styles, state, fonts, meta, links, modules, title}: Options) {
+function injectHTML(
+  book: ArchiveBook,
```

Comment:
you're already receiving state, you can put your selector in this function and not add the extra argument